src/common-principles.md

```diff
@@ -536,12 +550,39 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information such as physiological recordings MAY be stored with
+[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions).
+Rules for formatting plain-text tabular files apply to TSVGZ files with three exceptions:
+
+1.  The contents of TSVGZ files MUST be compressed with
+    [gzip](https://datatracker.ietf.org/doc/html/rfc1952).
+1.  Compressed tabular files MUST NOT contain a header in the first row
+    indicating the column names.
+1.  TSVGZ files MUST be accompanied by a JSON file with the same name as their
+    corresponding tabular file but with a `.json` extension.
+
+
+!!! note "Column definitions in compressed tabular files"
```

Comment:
@effigies - if the title is not going to be long, I'd simplify even more:
```suggestion
!!! warning "Attention"
```
and for this one, I'd re-up it to warning because this is a common friction point for newcomers.